build.sh

```diff
@@ -170,60 +164,22 @@ function docker_build_with_version {
 
   pull_image "$dockerfile"
 
+  is_podman() {
+    local _bin
+    _bin=$(docker --version | cut -d" " -f1)
```

Comment:
This will work with `podman-docker`, but not when a symlink is used (eg. `/usr/bin/docker -> podman`) on systems where podman-docker might not be available. Not sure if we want to make sure this case is covered as well, but mentioning anyway.

Reply:
I really did not expect this behavior. I will try to rethink this, so that also `/usr/bin/docker -> podman` is usable.